### DISCUSSION TITLE
Como exibir conteúdo do site conforme o histórico de navegação do usuário!

### DISCUSSION THREAD

**Discussion by lucas-oliveira-sympla** [frontendbr/forum#888] [Discussão]
Galera,

Gostaria de uma dica ou caminho de como exibir certas informações do meu site conforme o historico de acesso do cliente!

**Comment by JotaPe** [comment]
Entao que eu saiba (desculpa a falta de acentos teclado americano) acessar o historico de navegacao e controle da privacidade do usuario, e que eu tambem saiba navegadores nao dao acesso aberto ao historico do usuario, uma maneira e vc criar cookies para criar um "historico" do usuario.

**Comment by JotaPe** [comment]
Ter acesso ao historico de navegacao seria infrigir a privacidade do usuario.

**Comment by giovannism20** [comment]
@JotaPe ele quer algo como o Facebook faz, oferecer coisas baseado em locais por onde a pessoa navegou.

**Comment by TiagoDanin** [comment]
Como foi citado acima, solução de uso de cookie é uma boa. Mas há ferramentas que podem facilitar a telemetria, como o Teste A/B, recomendo da uma olhada aqui #856

**Comment by isaacguerreir** [comment]
Se você pode usar algumas bibliotecas de monitoramente de atividade do usuário na sua página como fonte de dados para modificar o conteúdo.

https://github.com/keen/common-web
